Examples.md

```diff
@@ -3,3 +3,21 @@ You want to see the examples? Did you even read all the features of DreamBerd? O
 
 Please go back to the features page by following [this link](https://github.com/TodePond/DreamBerd/blob/main/README.md).<br>
 When you have finished reading the features, you can find the _real_ examples [here](https://github.com/TodePond/DreamBerd/blob/main/res/Examples.md).
+
+```DREAMBERD PUBLIC LICENSE
+Version 1, April 2023
+
+Copyright (C) 2023 Lu Wilson <l2wilson94@gmail.com>
+
+Everyone is permitted to release software, as long as they follow the terms and conditions listed below.
+
+            DREAMBERD PUBLIC LICENSE
+   TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION, CREATION AND MODIFICATION
+
+ 0. By using this License, you acknowledge that any accompanying software is NOT owned by the DreamBerd Foundation.
```

Comment:
In the interest of keeping number bullet points consistent with how indexing works in DreamBerd, shouldn't this be -1 and the next entry be 0?